syntax = "proto3";

message CraneCommand {
    float command = 1; //e.g start
}